model/hyperlink_constants.py

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+class HyperlinkConstants(object):
```

Comment:
Let's call this `LinkRelations` -- it's easier to remember and it's a good match with `MediaTypes`.